src/main/java/de/dennisguse/opentracks/util/UnitConversions.java

```diff
@@ -51,6 +51,8 @@
     public static final double MM_TO_M = 0.001;
 
     // Distance
+    // multiplication factor to convert miles to meters
+    public static final double MI_TO_M = 1609.344;
```

Comment:
Isn't this: 
Can you introduce 
`
public static final double MI_TO_KM = 1 / KM_TO_MI;
public static final double MI_TO_M = MI_TO_KM * KM_TO_M;
`

PS: Did not test this.....

Reply:
> Can you introduce
> public static final double MI_TO_KM = 1 / KM_TO_MI; public static final double MI_TO_M = MI_TO_KM * KM_TO_M;

:+1: 

> PS: Did not test this.....

I check it out and I'll add tests if not any.